templates/home/index.html

```diff
@@ -417,7 +417,7 @@ <h1 data-aos="fade-up">Contact Us</h1>
         </div>
     </section>
     <div class="con-sidenav">
-        <a href="#contact-us">Contact Us</a>
+        <a href="#contact-us"><i class="fa-rotate-90 fas fa-phone"></i>Contact Us</a>
```

Comment:
Can you please put `fa-rotate-90` at the end of classes instead of at the start?